Source/ACE/Network/GameMessages/Messages/GameMessageAutonomousPosition.cs

```diff
@@ -18,6 +19,8 @@ public GameMessageAutonomousPosition(WorldObject worldObject)
                 Writer.Write(worldObject.Sequences.GetCurrentSequence(SequenceType.ObjectTeleport)); // teleport_timestamp - always 0 in my pcaps
                 Writer.Write(worldObject.Sequences.GetCurrentSequence(SequenceType.ObjectForcePosition)); // force_position_timestamp - always 0 in my pcaps
                 Writer.Write(1u); // contact - always "true" / 1 in my pcaps
+                if (p.Statemachine.CurrentState == (int)MovementStates.Idle)
+                    p.Statemachine.ChangeState((int)MovementStates.Moving);
             }
```

Comment:
again, if we make StateMachine private, you force this logic into the player class where it belongs.